### DISCUSSION TITLE
Custom values for language card

### DISCUSSION THREAD

**Discussion by Virgile-Foussereau** [anuraghazra/github-readme-stats#1862] [Ideas]
Hi :wave: 

I would like to use the language card on my profile but also on other places such as my CV. However a lot of my projects are not on github so I would like to create a language card with custom values that I could set to truly represents the languages I am using the most. Is this possible in any way?

Thanks!

**Comment by rickstaa** [comment]
Hey @Virgile-Foussereau. I like your idea, given that the language card is far from perfect (see #1801). The closest thing we currently have is #1600, which will allow users to skew the language card results. I think implementing a new `custom` argument would be even better. I converted this discussion into a feature request so that we can track the interest from the community.

**Reply by Virgile-Foussereau** [reply]
Awesome ! Keep me up to date if it is ever available !
